src/lib/hostapd.rb

```diff
@@ -123,7 +123,8 @@ def self.interface_name(networks)
 
   def self.password_for_network(network)
     path = File.join network[:path], 'password'
-    File.read path if File.readable? path
+    password = File.read path if File.readable? path
+    password.strip
```

Comment:
Better version:

```
File.read(path).strip if File.readable? path
```

This way you get the stripped value and nil if path is not readable. In the current version you will get `NoMethodError` on (effectively) `nil.strip`